addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -23,10 +23,15 @@
                 <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
                     <t t-esc="attachmentViewer.attachment.displayName"/>
                 </div>
+                <div class="o-autogrow"/>
                 <div class="o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="_onClickDownload" role="button" title="Download">
-                    <i class="fa fa-download fa-fw" role="img"/>
+                    <button class="btn btn-sm btn-primary o_AttachmentViewer_headerItemButtonPrimary" t-att-class="{'o-mobile': env.messaging.device.isMobile}" data-toggle="tooltip">
```

Comment:
Isn't it possible to make it work without adding all this extra button stuff? What is the goal here?

Reply:
The goal of this commit is to improve the ux when an user display a document. Currently we have this (on mobile device): https://i.imgur.com/m4A8N2A.png with this commit we move to https://i.imgur.com/FxgHG96.png
I will rework the commit message to add explanation around that and apply to the div `o_AttachmentViewer_buttonDownload` the visual button look.